index.html

```diff
@@ -84,7 +85,7 @@ <h1 class="logo"><a href="#">ProfNITT</a></h1>
       <div class="row justify-content-center">
         <div class="col-xl-8">
           <h1>ProfNITT</h1>
-          <h2>The Finance and Investment club of NIT Trichy.</h2> 
+          <h2>The Finance and Investment club of NIT Trichy.</h2>

```

Comment:
Not relevant to the scope of this PR.

Reply:
I added an item to the drop down for Case Solving Competitions, hence this shift by a line. Ignore it?

Reply:
No, see the change - you have deleted the space after `</h2>`. While it makes sense to do it, it is outside the scope of this PR.

Reply:
cool just let it be for now, and dont repeat it henceforth then?

Reply:
Bump.

Reply:
@9401adarsh